broker-plugin/broker-cli/src/main/java/io/enmasse/broker/cli/Cli.java

```
@@ -0,0 +1,105 @@
+import java.util.Arrays;
+import java.util.concurrent.TimeoutException;
+
+public class Cli implements Closeable {
+
+    private static final Logger log = LoggerFactory.getLogger(Cli.class.getName());
+    private SyncRequestClient brokerClient;
+    private Artemis artemis;
+    private String command;
+    private String queue;
+
+
+    private boolean parserArgs(String[] args) {
+
+        CLI parser = new DefaultCLI().setName("broker-cli.jar");
+        parser.addOption(new Option()
+            .setLongName("command")
+            .setShortName("C")
+            .setRequired(true)
+            .addChoice("purgeQueue")
+            .addChoice("getMessageCount")
+            .setDescription("Command to run on broker"));
+        parser.addOption(new Option()
+            .setLongName("queue")
+            .setShortName("Q")
+            .setRequired(true)
+            .setDescription("Queue to run command against"));
+
+        StringBuilder builder = new StringBuilder();
+        parser.usage(builder);
+
+        try {
+            CommandLine cl = parser.parse(Arrays.asList(args));
+            this.command = cl.getOptionValue("C");
+            this.queue = cl.getOptionValue("Q");
+            return true;
+        } catch (Exception e) {
+            log.info("\n" + builder.toString());
+        }
+        return false;
+    }
+
+    private void start() throws Exception {
+
+        Vertx vertx = Vertx.vertx();
+        ClientFactory clientFactory = new ClientFactory(vertx);
+        brokerClient = clientFactory.connectBrokerManagementClient("localhost", 5673);
+        artemis = new Artemis(brokerClient);
+    }
+
+    private void run() throws TimeoutException {
+
+        switch (command) {
+            case "getMessageCount":
+                log.info("Count: " + artemis.getQueueMessageCount(queue));
+                break;
+            case "purgeQueue":
+                artemis.purgeQueue(queue);
+                break;
+        }
```

I see you've configured the cli parsing to allow only these possibilities, but I'd still add a default: label with a  throw IllegalStateException if command is somehow unexpected.